### ISSUE TITLE
Cannot find name 'ref'

### ISSUE THREAD

**Issue by Diokuz** [manifoldco/swagger-to-ts#265]
I have an OpenAPI json like that:
```json
{
  "swagger": "2.0",
  "definitions": {
    "Messages": {
      "type": "object",
      "additionalProperties": {
        "$ref": "#/definitions/Message"
      }
    },
    "Message": {
      "type": "object",
      "properties": {
        "code": {
          "type": "integer"
        },
        "text": {
          "type": "string"
        }
      }
    }
  }
}
```

Which results in typescript:

```ts
export interface definitions {
  Messages: { [key: string]: ref }
  Message: { code?: number; text?: string }
}
```

It is not valid, because of `ref`. What is `ref`, and why it is not defined?

**Comment by Diokuz** [comment]
AFAIK should be
```ts
export interface definitions {
  Messages: { [key: string]: definitions['Message'] }
  Message: { code?: number; text?: string }
}
```

**Comment by Diokuz** [comment]
Looks like it was broken here https://github.com/manifoldco/swagger-to-ts/pull/223/files#diff-92e97370f1732d8782fd35b2cfaf277bL59

**Comment by wmonk** [comment]
Also facing this issue. Any ideas on a timeframe for a fix?

**Comment by pedrotorchio** [comment]
Same issue

**Comment by lukeshiru** [comment]
Same problem, here!

**Comment by jcf120** [comment]
Yep, same here

**Comment by drwpow** [comment]
Fixed (hopefully) in #266
